frontend/src/Components/Home/PopularArticles/ArticleImage/ArticleImage.jsx

```diff
@@ -0,0 +1,14 @@
+import { Image } from "react-bootstrap";
+import "./ArticleImage.scss";
+import NumberArrowMarker from "../NumberArrowMarker/NumberArrowMarker";
+
+export default function ArticleImage({ article_img, number }) {
+  return (
+    <>
+      <div>
+        <NumberArrowMarker number={number}/>
+        <Image src={article_img} className="article-img" />
```

Comment:
give a more specific class name because i feel like this may be reused elsewhere in the future